source/routing/raptor_api.cpp

```diff
@@ -1124,14 +1124,30 @@ boost::optional<routing::map_stop_point_duration> get_stop_points(const type::En
     return result;
 }
 
-static boost::optional<routing::map_stop_point_duration> get_stop_points_if_not_already_done(
+static routing::map_stop_point_duration make_map_stop_point_duration(
+    const std::vector<type::EntryPoint>& entryPointList,
```

Comment:
It would save you loads of writing by tydefing:
```cpp
using EntryPoints = std::vector<type::EntryPoint>;
```
It's also much better if we decide later on to change the container type ;)